server/controllers/request.js

```diff
@@ -6,7 +6,7 @@ const getRequestsByUser = (req, res, next) => {
     return next(createError(403));
```

Comment:
Just a minor note that you don't have to check for req.user. Passport will automatically throw a 403 Forbidden if user is unauthenticated.

Reply:
And thanks for prettifying my code lol

Reply:
@dproc96 small change for you before merging. Thanks Michael!